pkg/mounter/refcounter_windows.go

```diff
@@ -0,0 +1,105 @@
+//go:build windows
+// +build windows
+
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package mounter
+
+import (
+	"crypto/md5"
+	"fmt"
+	"os"
+	"strings"
+	"sync"
+)
+
+var basePath = "/csi/mounts"
```

Comment:
rename `mounts` to `smbmounts`, 
the windows path should be `c:\csi\smbmounts`?

Reply:
Yep. c:\csi is the path where `plugin-dir` volume mounts (hostpath: C:\var\lib\kubelet\plugins\smb.csi.k8s.io\)
I will change it to windows style.

Reply:
done